common/NuprojInitializer.msbuildproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Please keep 15.0

---
In reply to: [107542980](https://github.com/Microsoft/WinObjC/pull/2300#discussion_r107542980) [](ancestors = 107542980)